joern-cli/frontends/gosrc2cpg/src/main/scala/io/joern/gosrc2cpg/astcreation/AstForExpressionCreator.scala

```diff
@@ -23,24 +23,14 @@ trait AstForExpressionCreator(implicit withSchemaValidation: ValidationMode) { t
     }
   }
 
-  private def astForStructType(expr: ParserNodeInfo): Seq[Ast] = {
```

Comment:
@hiteshbedre We can move this method as well to as this related to TypeDecl `AstForStructuresCreator.scala`